docs/DeveloperGuide.adoc

```diff
@@ -261,45 +261,50 @@ image::UML REST API.png[width="400"]
 
 Including UI features to make Outlook's mail drafting screen appear in the `Browser Panel`, and requiring the user to input the index of student to email, the subject, and the body as three seperate inputs instead of one.
 
-//tag::schedule[]
-=== Schedule feature
+//tag::event[]
+=== Event feature
 ==== Current Implementation
 
-The schedule command is a completely new feature that allows the user to schedule events within ConTAct.
-It follows a similar command flow as that of the address book, but it will have it's own separate model and
-.xml file for storage. Furthermore, to facilitate this command, a new data type is introduced: the Event class.
-This Event class has it's own unique fields that the user can specify to schedule an event:
+Th event feature is largely centered around the `Event` class, and is integrated into ConTAct in a similar fashion to the address book.
+The main commands supporting this feature is `schedule` and `cancel`, which both interact with the `Calendar` model.
```

Comment:
commands supporting this feature ~~is~~ are

Reply:
fixed typo, thanks!